test/fixtures/file.csv

```diff
@@ -0,0 +1,27 @@
+headercolumn_0,headercolumn_1,headercolumn_2,headercolumn_3,headercolumn_4,headercolumn_5,headercolumn_6,headercolumn_7,headercolumn_8,headercolumn_9
```

Comment:
can we make this simpler ?

Reply:
How do you mean? I'm happy to make the file more reasonable. My intent is to ensure we have enough data in the file to raise the unwanted exception.

Any suggestions?